content/guides/getting-started.md

```diff
@@ -0,0 +1,235 @@
+
+``` diff
+webpack-demo
+|- package.json
++ |- dist
++   |- index.html
+- |- index.html
+|- /src
+  |- index.js
+```
+
+To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally...
+
+```bash
+npm install --save lodash
+```
+
+and then import it from our script...
+
+__src/index.js__
+
+```diff
++ import _ from 'lodash';
+
+function component () {
+// ...
+```
+
+We also need to change our update our `<script>` tags to point to the unified bundle:
+
+__dist/index.html__
+
+```diff
+ <html>
+  <head>
+    <title>webpack 2 demo</title>
+-   <script src="https://unpkg.com/lodash@4.16.6"></script>
+  </head>
+  <body>
+-   <script src="src/index.js"></script>
++   <script src="dist/bundle.js"></script>
+  </body>
+ </html>
+```
+
+In this setup, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution). By stating what dependencies a module needs, webpack can use this information to build a dependency graph. It then uses the graph to generate an optimized bundle where scripts will be executed in the correct order.
+
+With that said, let's run `webpack` with our script as the [entry point](/concepts/entry-points) and `bundle.js` as the [output](/concepts/output):
+
+```bash
+./node_modules/.bin/webpack src/index.js dist/bundle.js
+
+Hash: ff6c1d39b26f89b3b7bb
+Version: webpack 2.2.0
+Time: 385ms
+    Asset    Size  Chunks                    Chunk Names
+bundle.js  544 kB       0  [emitted]  [big]  main
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 278 bytes {0} [built]
+```
+
+T> Output may vary. If the build is successful then you are good to go.
+
+Open `index.html` in your browser and, if everything went right, you should see the following text: 'Hello webpack'.
+
+
+## ES2015 Modules
+
+Notice the use of [ES2015 module `import`](https://developer.mozilla.org//en-US/docs/Web/JavaScript/Reference/Statements/import) (alias ES2015, *harmony*) in `src/index.js`? Although `import`/`export` statements are not supported in browsers (yet), webpack supports them and will replace those instructions with ES5 compatible wrapper code in the output. Inspect `dist/bundle.js` if you need to assure yourself.
+
+Note that webpack will not alter any code other than `import` and `export` statements. In case you are using other [ES2015 features](http://es6-features.org/), make sure to use a transpiler such as [Babel](https://babeljs.io/) or [Bublé](https://buble.surge.sh/guide/). See our [Module API](/api/module-methods) documentation for information on the various module syntaxes supported by webpack.
+
+
+## Using a Configuration
+
+For a more complex setup, we can use a webpack [configuration file](/concepts/configuration). Let's add this file and represent the CLI command used above with some basic configuration:
```

Comment:
Suggestion: "You can also create a [configuration file](/concepts/configuration) that webpack can use when you run webpack. This is much more efficient than having to type in a lot of commands in the terminal, so let's create one for our basic setup:"